dash_auth/oauth.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import absolute_import
+import datetime
+import flask
+from flask_seasurf import SeaSurf
+import json
+import os
+from .auth import Auth
+
+
+class OAuthBase(Auth):
+    AUTH_COOKIE_NAME = 'plotly_auth'
```

Comment:
Should this be `plotly_auth` in the base class?

Reply:
How about something like `dash_token`?